_pages/theming.md

```diff
@@ -1,104 +1,100 @@
 
-# How to customize the Sass stylesheets
+### Individual paragraphs
+
+The templates used to generate paragraphs can be replaced selectively, which is useful if you want to emphasize a particular paragraph or add links to external sources that don’t exist in the regulation itself. Note that this mechanism is intended for one-offs; consider modifying the data structures instead if you find yourself using it often.
 
-This section explains eRegulations theming from the Sass/CSS perspective.
+To use this override mechanism, create a `templates/regulations/custom_nodes` directory in your Django application if it doesn't already exist. Inside that folder, create files corresponding to node labels, such as, `478-103-b.html`. These templates will be used **in place** of the `tree-with-wrapper.html` template, so be sure to provide the functionality already present there. Should you need to use this functionality only on specific versions, your template can make use of the `version` context variable.
 
-The base theme is heavily parameterized to allow for customization. It’s designed to be customized through the use of sass variables (`_variables.scss`) and mixins (`_custom.scss`), particularly font mixins.
+## Customize the stylesheets
 
-This is designed so that you can build "overriding" files in an agency-specific wrapper repository, instead of directly modifying the base files in **regulations-site**. For an example (but using Less), see [these .less files in **atf-eregs**](https://github.com/18F/atf-eregs/tree/master/atf_eregs/static/regulations/css/less).
+The base theme is designed to be customized with [Sass](http://sass-lang.com/) variables (`_variables.scss`) and mixins (`_custom.scss`), particularly font mixins.
+You can build "overriding" files in an agency-specific wrapper repository, instead of directly modifying the base files in **regulations-site**. For an example (using Less), see [these .less files in **atf-eregs**](https://github.com/18F/atf-eregs/tree/master/atf_eregs/static/regulations/css/less).
 
 ### Font mixins
 
-You can do most font theming by including the basic font mixins to create specific formatting rules for different types of content. Here’s an example that declares a basic font and then creates a variation with a particular size and line height to use for a specific type of content within the theme:
+You can do most font theming by including the basic font mixins to create specific formatting rules for different types of content. Here’s an example that declares a basic font and then creates a variation with a particular size and line height for a specific type of content within the theme:
 
-``` scss
-@mixin sans-font-regular {
-  font-family: "Source Sans Pro", Arial, sans-serif;
-  font-weight: 400;
-  font-style: normal;
-}
+``` scss  
```

Comment:
I am having the same issue, but not sure why it's not working. I repasted the code from the live site to fix the spacing. Do you want to go ahead and merge this, and assume it's a local jekyll or gulp issue?